src/main/java/com/patternpedia/api/entities/user/UserEntity.java

```diff
@@ -69,17 +83,23 @@
 //    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
why are the comments deleted?

Reply:
Which comments are deleted?

Reply:
why is there a '//' on this line? can we delete the line or will we need it in the future?